### DISCUSSION TITLE
How to get Input Action CallBack in MRTK3?

### DISCUSSION THREAD

**Discussion by SangonomiyaSakunovi** [MixedRealityToolkit/MixedRealityToolkit-Unity#673] [Q&A]
## Describe the issue
I want to use Azure Spatial Anchors or World Locking Tools or other modules, but the question is: How to get the Input by MRTK3?Please tell me how to do? Or can you let me know in which repo has some reference?
![mrtk3Input](https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/assets/119163281/da6b8859-d75c-4ad5-af1b-1cb7e55c220d)

## Feature area
In this document, there is nothing mentioned about how to add a call back. I know that in the XR Controller Script has some Action can reference, but that seems not the pos where I can add my Action. 

## Existing doc link
[If this is about something in an existing document, please provide link](https://learn.microsoft.com/en-us/windows/mixed-reality/mrtk-unity/mrtk3-input/packages/input/interaction-mode-manager?source=recommendations)

**Comment by AMollis** [comment]
@SangonomiyaSakunovi  moving to our Q&A discussion
